al-jazari/roster.json

```diff
@@ -73,6 +73,10 @@
     {
 	"name": "Wing Yuen",
 	"github": "wingyuen2022"
-    }
+    },
+    {
+      "name": "Richard Sigl",
+      "github": "sigltech"
+    },    
```

Comment:
Can you please remove the trailing comma @sigltech :)?

Reply:
I think that should be resolved